src/modules/TeamsList/components/Teams/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { PageTitle } from 'typography';
+
+import { TeamsHeader } from './components/TeamsHeader';
```

Comment:
you can make index file in components folder, and reimport components like 
import { TeamsHeader, MyTeam, TeamItem } from './components';

Reply:
done